afl.pyx

```diff
@@ -75,10 +75,13 @@ def _hash(key, offset):
 
 cdef object trace
 def trace(frame, event, arg):
-    global prev_location
+    global prev_location, tstl_mode
     cdef unsigned int location, offset
+    cdef object filename = frame.f_code.co_filename
+    if tstl_mode and (filename[-7:] in ['sut.py','/sut.py']):
```

Comment:
Missing space after comma.

(I was going to complain that creating a list is more costly than creating a tuple, but it turns out it doesn't matter: Cython optimizes this into two equality comparisons anyway. The list syntax is more readable, so let's keep it.)